.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,28 @@
+name: test-deploy-gradle-sora-plugin
+on: [push]
```

Comment:
```suggestion
on: [pull_request]
```

Reply:
I guess it's on `push` to the `master` (latest) or explicit release versions corresponding to GH tags, we should add value setting for the version responsible env var